mappings/net/minecraft/block/entity/SignBlockEntity.mapping

```diff
@@ -24,3 +27,14 @@ CLASS net/minecraft/class_2625 net/minecraft/block/entity/SignBlockEntity
 		ARG 1 value
 	METHOD method_30843 getTextOnRow (IZ)Lnet/minecraft/class_2561;
 		ARG 1 row
+		ARG 2 filtered
+	METHOD method_33384 unparsedTextFromJson (Ljava/lang/String;)Lnet/minecraft/class_2561;
+		ARG 1 json
+	METHOD method_33827 setTextOnRow (ILnet/minecraft/class_2561;Lnet/minecraft/class_2561;)V
+		ARG 1 row
+		ARG 2 text
+		ARG 3 filteredText
+	METHOD method_33828 parseTextFromJson (Ljava/lang/String;)Lnet/minecraft/class_2561;
+		ARG 1 json
+	METHOD method_33830 getTexts (Z)[Lnet/minecraft/class_2561;
```

Comment:
```suggestion
	METHOD method_33830 getText (Z)[Lnet/minecraft/class_2561;
```

Reply:
Hmm, this returns an array of text

Reply:
The `text` field needs to be renamed to follow this method's name